lib/flagutil/map.go

```diff
@@ -0,0 +1,37 @@
+package flagutil
+
+import (
+	"flag"
+	"fmt"
+	"strings"
+)
+
+type MapString map[string]string
+
+func (m *MapString) String() string {
+	return fmt.Sprintf("%v", *m)
```

Comment:
if nill return "" ?

Reply:
yeah I guess I should handle null there was being a bit lazy.